FirebaseCombineSwift/Sources/Auth/GameCenterAuthProvider+Combine.swift

```diff
@@ -12,14 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if canImport(Combine) && swift(>=5.0)
+#if canImport(Combine) && swift(>=5.0) && !os(watchOS)
```

Comment:
nit: We should be able to drop `&& swift(>=5.0)` now that we require `5.9`.

Reply:
Good catch! Done.